src/models/components/stacked_hidden_state.py

```diff
@@ -0,0 +1,53 @@
+import torch.nn as nn
+from torch import Tensor
+
+
+class HiddenState(nn.Module):
+    def __init__(self, module: nn.Module, hidden_init: Tensor):
```

Comment:
ここは実際には `nn.Parameters`が入力されるのでアノテーションはこうなります。（ただし、nn.Parameters型はtorch.Tensor型のサブクラスである...)
```suggestion
    def __init__(self, module: nn.Module, hidden_init: nn.Parameters):
```

Reply:
https://github.com/MLShukai/PrimitiveAMI/pull/141/commits/12250ad4bbfa13101087d643853ede69206afbcc
修正しました